cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -51,13 +58,13 @@ enum class Radix : int32_t { BASE_2 = 2, BASE_10 = 10 };
 template <typename T>
 constexpr inline auto is_supported_representation_type()
 {
-  return std::is_same<T, int32_t>::value || std::is_same<T, int64_t>::value;
+  return simt::std::is_same<T, int32_t>::value || simt::std::is_same<T, int64_t>::value;
```

Comment:
Why use the `simt` version here?

Reply:
It is necessary to get the jitified version of the file to work for binaryops. @devavret is the one that helped me out with this.